packages/babel-plugin-transform-es2015-modules-commonjs/src/index.js

```diff
@@ -8,7 +8,27 @@ import {
 } from "babel-helper-module-transforms";
 import simplifyAccess from "babel-helper-simple-access";
 
-export default function({ types: t, template }) {
+export default function({ types: t, template }, options) {
+  const {
+    allowTopLevelThis,
+    loose,
+    noInterop,
+    strict,
+    strictMode,
+
+    // Defaulting to 'true' for now. May change before 7.x major.
+    allowCommonJSExports = true,
+  } = options;
+
+  const ref = {
```

Comment:
I personally prefer if this was moved back. I think there's a difference between hoisting option parsing and usage.

Reply:
+ we do not end up with `ref` variable which doesn't hold any meaningful info

Reply:
Alright, I've got your point